### PR TITLE
creation and working on form table app

### PR DESCRIPTION
**Pull request by AshwaniChauhan1**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png" >
-    <HelloWorld/>
+    <Form/>
   </div>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
+import Form from './components/Form.vue'
```
Comment (vueexcel) [inline src/App.vue]: use shorthand "@" while importing components
eg:- @/components/Form.vue

File: src/components/Form.vue [hunk 2]
```diff
@@ -0,0 +1,135 @@
+<template>
+    <div>
+        <div><h1 class="py-3">Fill Form</h1></div>
+        <div class="d-flex justify-content-center">
+            <b-form  style="width:40%;" v-on:submit.prevent >
```
Comment (vueexcel) [inline src/components/Form.vue]: don't use inline css

File: src/components/Form.vue [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+            val:{},
+            password:'',
+            cpassword:'',
+            status:"not_accepted",
+            editIndex:false,
+            indexval:''
+            }
+        },
+    methods:{
+        added(){
```
Comment (vueexcel) [inline src/components/Form.vue]: use proper function / method names

File: src/components/Form.vue [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+            else if(this.password != this.cpassword){
+                this.error='Password not match';
+            }
+            else if(this.status != "accepted"){
+                this.error='please read terms and conditions';
+            }
+            else{
+                if (this.editIndex ==true) {
+                //eslint-disable-next-line
+                console.log(this.items[this.indexval]);
```
Comment (vueexcel) [inline src/components/Form.vue]: remove consoles before pushing

File: src/components/Form.vue (hunk 3 above)
Comment (AshwaniChauhan1) [reply src/components/Form.vue]: done

File: src/components/Form.vue (hunk 4 above)
Comment (AshwaniChauhan1) [reply src/components/Form.vue]: done

File: src/components/Form.vue (hunk 2 above)
Comment (AshwaniChauhan1) [reply src/components/Form.vue]: done

File: src/App.vue (hunk 1 above)
Comment (AshwaniChauhan1) [reply src/App.vue]: done

File: src/components/Form.vue [hunk 5]
```diff
@@ -45,14 +45,31 @@
...
-        <template v-slot:cell(edit)="row">
-          <a @click="edit(row)">Edit</a>
-        </template>
-        <template v-slot:cell(delete)="row">
-          <a @click="del(row)">Delete</a>
-        </template>
-      </b-table>
+      <table class="table table-bordered">
+        <thead class="bg-light">
+          <tr>
```
Comment (vueexcel) [inline src/components/Form.vue]: can you use loop here

Comment (AshwaniChauhan1) [reply src/components/Form.vue]: no, if i use loop  here ,i cannot apply event on only  edit here

Comment (vueexcel) [reply src/components/Form.vue]: this is bad practice **items[0]** if we have 100's of items this wont be feasible

**Pull request merged**